editor/filesystem_dock.cpp

```diff
@@ -1883,8 +1883,8 @@ Vector<String> FileSystemDock::_check_existing() {
 	return conflicting_items;
 }
 
-void FileSystemDock::_move_dialog_confirm(const String &p_path) {
-	_move_operation_confirm(p_path, move_dialog->is_copy_pressed());
+void FileSystemDock::_move_dialog_confirm(const String &p_path, bool p_copy) {
+	_move_operation_confirm(p_path, p_copy);
```

Comment:
You could connect to this method directly. The intermediate one was only needed for getting the bool.

Reply:
The `connect()` does not support default argument. In order to connect to `_move_operation_confirm` directly, I have to bind `OVERRITE_UNDECIDED`. But in order to bind it, it seems I have to register the enum for binding. Since the class and enum is not exposed for scripting, I think doing this makes things more complicated.

Reply:
OK, I did the change. Note that `Overwrite` enum is now changed to public and bind.